### ISSUE TITLE
[Feature] Add option to add QR code to phone wallet

### ISSUE THREAD

**Issue by abdullah50053** [acmutd/hackportal#211]
# Overview

Currently, the QR code is only accessible through HackPortal. The QR code is static and does not change throughout the course of the hackathon.

# Proposal

Add an option to add the QR code to a phone's wallet. This includes Apple Wallet, Google Wallet, and Samsung Wallet.

**Comment by adityarathod** [comment]
Apple Wallet implementation will require an Apple Developer account, and passes need to be signed JSON bundles. [Documentation](https://developer.apple.com/documentation/walletpasses)

**Comment by legitatx** [comment]
> Apple Wallet implementation will require an Apple Developer account, and passes need to be signed JSON bundles. [Documentation](https://developer.apple.com/documentation/walletpasses)

Yeah, I've discussed it with @abdullah50053, and he's looking into setting up an account for ACM atm, but docs will also include this information for others.
